packages/block-library/src/table-of-contents/save.js

```diff
@@ -2,19 +2,23 @@
  * WordPress dependencies
  */
 import { useBlockProps } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import TableOfContentsList from './list';
 import { linearToNestedHeadingList } from './utils';
 
-export default function save( { attributes: { headings = [] } } ) {
+export default function save( { attributes: { headings = [], ariaLabel } } ) {
 	if ( headings.length === 0 ) {
 		return null;
 	}
 	return (
-		<nav { ...useBlockProps.save() }>
+		<nav
+			{ ...useBlockProps.save() }
+			aria-label={ ariaLabel || __( 'Table of Contents' ) }
+		>
```

Comment:
```suggestion
		<nav { ...useBlockProps.save() }>
```

We don't need to update the save function, as block support should handle the attributes. By the way, please don't use translation functions in the save function, as the block will break when the site's locale changes.

Reply:
I've removed the update to save.js in https://github.com/WordPress/gutenberg/pull/71586/commits/15848d446f22bd55bc3a2ae4e1e85ea6834ac162.

(And with my latest updates, removed the save function altogether 😅 )